openpype/hosts/nuke/startup/custom_popup.py

```diff
@@ -0,0 +1,182 @@
+import sys
+import contextlib
+import re
+import nuke
+from PySide2 import QtCore, QtWidgets
+
+
+def get_main_window():
+    """Acquire Nuke's main window"""
+    main_window = None
+    if main_window is None:
+
+        top_widgets = QtWidgets.QApplication.topLevelWidgets()
+        name = "Foundry::UI::DockMainWindow"
+        for widget in top_widgets:
+            if (
+                widget.inherits("QMainWindow")
+                and widget.metaObject().className() == name
+            ):
+                main_window = widget
+                break
+    return main_window
+
+
+class CustomScriptDialog(QtWidgets.QDialog):
+    """A Popup that moves itself to bottom right of screen on show event.
```

Comment:
Where is the logic that does this? Are we sure it's doing that?

If we would want it to do then then what's the reason we can't use the `openpype.widgets.popup.PopUp` class?

I feel like we should be working to having unified toast message pop-ups across all hosts whenever possible. So maybe there's a way that we can turn this into a dedicated Popup too or use a base class of sorts? Because somehow this somewhat sounds like [it's inline with the design proposals that came up here](https://github.com/ynput/OpenPype/issues/1780#issuecomment-1091077123).

However if it's not intended like that then I suppose just this part of the docstring is wrong?